src/server/graphql/types/TeamMember.js

```diff
@@ -76,7 +75,7 @@ const TeamMember = new GraphQLObjectType({
       }
     },
     meetingMember: {
-      type: MeetingMember,
+      type: require('./MeetingMember').default,
```

Comment:
Why do this lazy?

Reply:
because it comes up undefined if i don't. See User.js, i end up having to do this SO much! once node handles imports natively it'll all be better...